### PR TITLE
Indy projects update

### PR DESCRIPTION
**Pull request by cgrigis**

### CODE REVIEW

**Comment by cgrigis** [comment]
Beware the `Prettier` pass. :)

File: data/INDY/projects.yaml [hunk 1]
```diff
@@ -14,16 +14,24 @@ projects:
...
+          - label: Published in
+            text: proceedings of NeurIPS 28, 2015
+            url: https://proceedings.neurips.cc/paper/2015
       - type: Paper
         title: "ChoiceRank: Identifying Preferences from Node Traffic in Networks"
         url: https://infoscience.epfl.ch/record/229164/files/choicerank.pdf
+        notes:
+          - label: Published in
+            text: proceedings of Machine Learning Research Volume 70, August 2017
+            url: https://proceedings.mlr.press/v70/
```
Comment (tharvik) [inline data/INDY/projects.yaml:27]: SSL error? Cert is only valid for github and co

Comment (ineiti) [reply data/INDY/projects.yaml:27]: yes, plain old http looks better there...

File: updates/2021-03.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+210305 - Linus:
```
Comment (ineiti) [inline updates/2021-03.md:1]: Imposing personal preference: please keep one file, `2021-02.md`, which reflects all changes done for the `maintenance-pass` started in February 2021. Alternatively, find a better name. E.g., `2021-maintenance-1.md` or so...

Comment (cgrigis) [reply updates/2021-03.md:1]: In the absence of other explanation, I had assumed that a file named `updates/2021-02.md` meant the updates done in February, I guess that was wrong...

**Review by tharvik — APPROVED** [review]

File: updates/2021-03.md (hunk 2 above)
Comment (ineiti) [reply updates/2021-03.md]: yes, that was a good guess! Sorry to induce you into error...

**Pull request merged**